zng/value.go

```diff
@@ -37,11 +37,11 @@ func Parse(v ast.Literal) (Value, error) {
 }
 
 //XXX b should be zcode.Bytes
```

Comment:
update comment

Reply:
Done in 511b3f7a9126cda1b047449e7e8068aae111f3a8.